src/jose/Converter.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+/**
+ * Module Dependencies
+ * @ignore
+ */
+const JWT = require('./JWT')
+const JWD = require('./JWD')
+
+/**
+ * Converter
+ */
+class Converter {
+
+  /**
+   * toJWD
+   *
+   * @description
+   * Convert a JWT to JWD
+   *
+   * @param  {JWT} data
+   * @return {JWD}
+   */
+  static toJWD (data) {
+    return new JWD(data)
+  }
+
+  /**
+   * toJWT
+   *
+   * @description
+   * Convert a JWD to JWT
+   *
+   * @param  {JWD} data
+   * @return {JWT}
+   */
+  static toJWT (data) {
+    return new JWT(data)
```

Comment:
Tell me more about the design decision of using a separate Converter class for these two (fairly simple) functions, instead of placing them directly on the JWT and JWD classes. 
I'm assuming this is a placeholder for future functionality?